src/components/search-field-container.js

```diff
@@ -75,9 +77,14 @@ class FederatedSearchFieldContainer extends React.Component {
   }
 }
 
+FederatedSearchFieldContainer.defaultProps = {
+  children: [],
+};
+
 FederatedSearchFieldContainer.propTypes = {
-  children: PropTypes.array,
-  onNewSearch: PropTypes.func,
+  children: PropTypes.arrayOf(PropTypes.object),
```

Comment:
I used `PropTypes.node` above, so I may need to change this. Also, I've used a default value here instead of `isRequired`. Whichever is correct, I'll make these consistent.